lib/output.js

```diff
@@ -787,6 +787,11 @@ function OutputStream(options) {
         }
     });
 
+    PARENS(AST_BigInt, function(output) {
+        var p = output.parent();
+        if (p instanceof AST_PropAccess && p.expression === this) return this.value[0] == "-";
```

Comment:
This shouldn't be produced by parse or any transform - BigInts and numbers can't be negative - they have Ast_UnaryPrefix of "-".  See: `echo '(-0xAB_123n).toString()' | uglify-js -o ast`

Reply:
Indeed &minus; in fact that shouldn't happen to `AST_Number` either.

Reply:
Removed checks for negative in `AST_BigInt` & `AST_Number` in 773b7db869b6b6eae01c44d97866f96479051bc9